tools/oot-driver/charts/iavf-driver-0.0.1/Dockerfile-driver.SRO

```diff
@@ -55,6 +55,7 @@ ENV KERNEL_VERSION=$KVER
 RUN dnf install -y kmod
 
 COPY --from=builder /lib/modules/$KVER/updates/drivers/net/ethernet/intel/iavf/iavf.ko /oot-driver/
+COPY --from=builder /lib/modules/$KVER/updates/drivers/net/ethernet/intel/auxiliary/auxiliary.ko /oot-driver/
```

Comment:
This might not work for old versions of the driver.

Reply:
is this driver always built?

Reply:
from ice version 1.7.7 and iavf version 4.3.12 the auxiliary module is built.